src/components/ImageCard.js

```diff
@@ -1,11 +1,42 @@
 import styled from 'styled-components';
+import { useState } from 'react';
+import DeleteMessage from './DeleteMessage';
+import IconButton from './IconButton';
+import ScreenReaderOnly from './ScreenReaderOnly';
+import { BsTrash } from 'react-icons/bs';
+
+export default function ImageCard({
+  title,
+  tradition,
+  photo,
+  onDeleteTradition,
+  _id,
+}) {
+  const [showMessage, setShowMessage] = useState(false);
 
-export default function ImageCard({ tradition, photo }) {
   return (
     <>
       <CardWrapper>
         <CardPhoto src={photo}></CardPhoto>
+        <CardTitle>{title}</CardTitle>
         <CardTradition>{tradition}</CardTradition>
+
+        <IconButton
+          type="button"
+          aria-label="delete this card"
+          _id={_id}
+          onClick={() => setShowMessage(true)}
+          variant="trash"
+        >
+          <BsTrash style={{ width: '25px', height: '25px' }} />
+          <ScreenReaderOnly>delete this card</ScreenReaderOnly>
+        </IconButton>
+        {showMessage && (
+          <DeleteMessage
+            onConfirmDelete={() => onDeleteTradition(_id)}
+            onCancelDelete={() => setShowMessage(false)}
+          />
+        )}
       </CardWrapper>
     </>
```

Comment:
You don't need the fractions here:
```suggestion
      <CardWrapper>
        <CardPhoto src={photo}></CardPhoto>
        <CardTitle>{title}</CardTitle>
        <CardTradition>{tradition}</CardTradition>

        <IconButton
          type="button"
          aria-label="delete this card"
          _id={_id}
          onClick={() => setShowMessage(true)}
          variant="trash"
        >
          <BsTrash style={{ width: '25px', height: '25px' }} />
          <ScreenReaderOnly>delete this card</ScreenReaderOnly>
        </IconButton>
        {showMessage && (
          <DeleteMessage
            onConfirmDelete={() => onDeleteTradition(_id)}
            onCancelDelete={() => setShowMessage(false)}
          />
        )}
      </CardWrapper>
```